### ISSUE TITLE
Add C++ codes in C++ folder.

### ISSUE THREAD

**Issue by rohansaini886** [rohansaini886/Hacktoberfest2022#4]
You can commit Cpp codes in the folder. But don't commit hello world.

**Comment by Hussain6** [comment]
Hi Sir can I also contribute to this repository?

**Comment by flow6979** [comment]
what type of code exactly do we need to add here? like some famous algorithm or solution to online coding problems. please clarify this as I am very much interested in contributing to this repository. thank you.

**Comment by rohansaini886** [comment]
Yes @flow6979 You can add the famous algorithm which are written in C++ language. If you want to add your online coding problems then you can add it in the 30daysofcode folder.

**Comment by Anchit-Nayak** [comment]
Hello I would love to contribute towards this project can you assign me as well?

**Comment by Anmol-Prasad** [comment]
How exactly to work for this sir and what kind of PRs are you expecting? Would love to work on this as well. 
Can you please brief us 💯

**Comment by rohansaini886** [comment]
@Anmol-Prasad You can add your add Your C++ codes for eg. Your mini projects, some algorithms implementation in CPP languages and some like this codes.

**Comment by omkarvw** [comment]
Do we need to be assigned an issue to be able to contribute?

**Comment by rohansaini886** [comment]
@omkarvw Assigning of an issue is not compulsory for making a contribution. You can contribute by making a pull request in this repo.

**Comment by Prajwal0225** [comment]
Hello, @rohansaini886  I recently started learning DSA. I am familiar with basic sorting and searching algorithms. So I'd like to add "C++ implementations" of those algorithms into this repo.

I would be putting the following Algorithm into action  ⤵️ 

Algorithm for Searching
1. Liner Search
2. Binary search 


Algorithm for Sorting
1. Bubble Sort
2. Insertion Sort
3. Selection Sort
4. Counting Sort
5. Merge Sort
6. Quick Sort 


So, could you please assign this problem to me so that I can get to work on it.

**Comment by rohansaini886** [comment]
@Prajwal0225 Yes you can add these codes in this tepo.

**Comment by Prajwal0225** [comment]
I will start working on it 😇.

**Comment by Shuvamrauniyar** [comment]
i would like to contribute on this and also i have good grip on cpp language.
please assign it to me.

**Comment by PravarAnu** [comment]
Hey @rohansaini886  I am a programmer and C++ is my prefferd language. I am here to contribute my codes related to algorithm written in c++. So, can you assign it to me under the HACKTOBER FEST 2022.

**Comment by mohansharma22** [comment]
hey @rohansaini886  what type of c++ code we have to add in c++ folder ? can we add famous algorithms or famous problem solution ?

**Comment by prachisuman25** [comment]
Hey, @rohansaini886 I want to contribute to this repository. So, can you please assign this issue to me too?
